### ISSUE TITLE
Organization validation

### ISSUE THREAD

**Issue by ionicc** [ionicc/Github-organization-inviter#9]
Validate the organization name before proceeding

**Comment by sbvkrishna** [comment]
I would like to work on this.
So, before creating an invitation, we need to verify the organisation name. Is that right? And should I use a package to do that?

**Comment by ionicc** [comment]
@sbvkrishna Yes correct, We need to validate if the user putting up the app has put a correct org name.
Also, If you're taking up this. You could take a look towards #8 as both of these are pretty sequential 😄
